app/controllers/food_requests_controller.rb

```diff
@@ -0,0 +1,24 @@
+class FoodRequestsController < ApplicationController
+
+  def create
+
+    current_user.food_requests.where("created_at BETWEEN ? AND ?", Date.today, Time.now).each do |i|
```

Comment:
Something feels wrong here. 
I ran an `rails c` query like this:
```
2.5.1 :005 > (Date.today .. Time.now).include?(Time.new("2018-08-29 00:01"))
 => false 
```
Essentially, you seem to be comparing `Date.today` (which doesn't have a hour, like 00:01) with a full `DateTime` object (date & hour), which may not work.

Use `DateTime.now.beginning_of_day` instead :)